### DISCUSSION TITLE
Change SMTP Server

### DISCUSSION THREAD

**Discussion by kylhuk** [simple-login/app#1232] [Q&A] [answered]
Hey SL Community

When setting up the SL server, I have used a domain with ".xyz" as top level domain. Unfortunately, I just realized that tools like SpamAssassin discriminate certain TLD (a known issue to SpamAssassin community). Since I have configured like 10 domains for SimpleLogin, I would like to switch to a different domain, which has a different TLD.

What do I need to do to change the SMTP from the initially configured one, to a new one?

**Comment by nguyenkims** [comment, marked as answer]
@kylhuk you can change the `EMAIL_DOMAIN` config that's used to sending forward emails. The new domain also should have SPF, DKIM, DMARC correctly set up to decrease a chance of emails being put in Spam. After that you can just restart the SimpleLogin containers for the change to be taken into account.

**Reply by kylhuk** [reply]
Okay, that did it and additionally I changed the domains in the postfix config `main.cf`. Thanks for the help, @nguyenkims
